Index: devel/cmake/Makefile
===================================================================
--- devel/cmake/Makefile	(revision 371644)
+++ devel/cmake/Makefile	(working copy)
@@ -18,7 +18,6 @@
 CONFIGURE_ARGS=	--prefix=${PREFIX} \
 		--datadir="/${DATADIR_REL}" \
 		--docdir="/${DOCSDIR_REL}" \
-		--sphinx-man \
 		--system-libs \
 		--parallel=${MAKE_JOBS_NUMBER} \
 		--init="${PATCHDIR}/InitialCache.cmake"
@@ -32,7 +31,6 @@
 OPTIONS_SUB=	yes
 HAS_CONFIGURE=	yes
 
-BUILD_DEPENDS=	sphinx-build:${PORTSDIR}/textproc/py-sphinx
 LIB_DEPENDS=	libcurl.so:${PORTSDIR}/ftp/curl \
 		libexpat.so:${PORTSDIR}/textproc/expat2
 RUN_DEPENDS=	${LOCALBASE}/share/cmake/Modules/CMake.cmake:${PORTSDIR}/devel/cmake-modules
Index: devel/cmake/pkg-plist
===================================================================
--- devel/cmake/pkg-plist	(revision 371644)
+++ devel/cmake/pkg-plist	(working copy)
@@ -2,23 +2,6 @@
 bin/cmake
 bin/cpack
 bin/ctest
-man/man1/ccmake.1.gz
-man/man1/cmake.1.gz
-man/man1/cpack.1.gz
-man/man1/ctest.1.gz
-man/man7/cmake-buildsystem.7.gz
-man/man7/cmake-commands.7.gz
-man/man7/cmake-developer.7.gz
-man/man7/cmake-generator-expressions.7.gz
-man/man7/cmake-generators.7.gz
-man/man7/cmake-language.7.gz
-man/man7/cmake-modules.7.gz
-man/man7/cmake-packages.7.gz
-man/man7/cmake-policies.7.gz
-man/man7/cmake-properties.7.gz
-man/man7/cmake-qt.7.gz
-man/man7/cmake-toolchains.7.gz
-man/man7/cmake-variables.7.gz
 share/aclocal/cmake.m4
 %%DATADIR%%/Help/command/FIND_XXX.txt
 %%DATADIR%%/Help/command/FIND_XXX_MAC.txt
Index: dns/bind910/Makefile
===================================================================
--- dns/bind910/Makefile	(revision 371644)
+++ dns/bind910/Makefile	(working copy)
@@ -25,13 +25,10 @@
 CPE_UPDATE=	${ISCVERSION:C/.*-//:tl}
 .endif
 
-LIB_DEPENDS=	libxml2.so:${PORTSDIR}/textproc/libxml2
-
 GNU_CONFIGURE=	yes
 CONFIGURE_ARGS=	--localstatedir=/var --disable-linux-caps \
 		--disable-symtable \
-		--with-randomdev=/dev/random \
-		--with-libxml2=${LOCALBASE}
+		--with-randomdev=/dev/random
 ETCDIR=		${PREFIX}/etc/namedb
 
 CONFLICTS+=	bind9*-9.[456789].* bind9*-sdb-9.[456789].*
Index: multimedia/ffmpegthumbnailer/Makefile
===================================================================
--- multimedia/ffmpegthumbnailer/Makefile	(revision 371644)
+++ multimedia/ffmpegthumbnailer/Makefile	(working copy)
@@ -18,12 +18,11 @@
 
 CPPFLAGS+=	-I${LOCALBASE}/include
 LIBS+=		-L${LOCALBASE}/lib
-CONFIGURE_ARGS=	--with-pkgconfigdir=${PREFIX}/libdata/pkgconfig --enable-gio
+CONFIGURE_ARGS=	--with-pkgconfigdir=${PREFIX}/libdata/pkgconfig --disable-gio
 
 GNU_CONFIGURE=	yes
 INSTALL_TARGET=	install-strip
 USES=		libtool pathfix pkgconfig
-USE_GNOME=	glib20
 USE_LDCONFIG=	yes
 
 OPTIONS_DEFINE=	GNOME MATE
Index: sysutils/ipmitool/Makefile
===================================================================
--- sysutils/ipmitool/Makefile	(revision 371644)
+++ sysutils/ipmitool/Makefile	(working copy)
@@ -13,10 +13,11 @@
 LICENSE=	BSD3CLAUSE
 LICENSE_FILE=	${WRKSRC}/COPYING
 
-USES=		tar:bzip2 gmake readline
+USES=		tar:bzip2 gmake
 USE_OPENSSL=	yes
 GNU_CONFIGURE=	yes
 CONFIGURE_ARGS+=--enable-intf-lan --enable-intf-lanplus \
+		--disable-ipmishell \
 		--disable-intf-imb --disable-intf-lipmi --disable-intf-bmc
 PLIST_FILES=	bin/ipmitool man/man1/ipmitool.1.gz man/man8/ipmievd.8.gz \
 		sbin/ipmievd share/ipmitool/oem_ibm_sel_map \
Index: sysutils/nut/Makefile
===================================================================
--- sysutils/nut/Makefile	(revision 371644)
+++ sysutils/nut/Makefile	(working copy)
@@ -60,6 +60,7 @@
 		--with-pkgconfig-dir=${PREFIX}/libdata/pkgconfig \
 		--with-user=${NUT_USER} \
 		--with-group=${NUT_GROUP} \
+		--with-wrap \
 		--with-dev
 
 .if ${PORT_OPTIONS:MCGI}
Index: lang/python27/Makefile
===================================================================
--- lang/python27/Makefile	(revision 372460)
+++ lang/python27/Makefile	(working copy)
@@ -14,7 +14,7 @@
 
 LICENSE=	PSFL
 
-USES=		cpe ncurses pkgconfig readline tar:xz
+USES=		cpe ncurses pkgconfig tar:xz
 USE_LDCONFIG=	yes
 USE_OPENSSL=	yes
 GNU_CONFIGURE=	yes
Index: lang/python27/pkg-plist
===================================================================
--- lang/python27/pkg-plist	(revision 372460)
+++ lang/python27/pkg-plist	(working copy)
@@ -2186,7 +2186,6 @@
 lib/python2.7/lib-dynload/ossaudiodev.so
 lib/python2.7/lib-dynload/parser.so
 lib/python2.7/lib-dynload/pyexpat.so
-lib/python2.7/lib-dynload/readline.so
 lib/python2.7/lib-dynload/resource.so
 lib/python2.7/lib-dynload/select.so
 lib/python2.7/lib-dynload/strop.so
Index: net/minidlna/Makefile
===================================================================
--- net/minidlna/Makefile	(revision 373800)
+++ net/minidlna/Makefile	(working copy)
@@ -21,10 +21,12 @@
 
 GNU_CONFIGURE=	yes
 CONFIGURE_ARGS=	--with-db-path=/var/db/minidlna \
+		--enable-thumbnail \
+		--with-os-version="CURRENT" \
 		--with-os-url="http://www.FreeBSD.org"
 CPPFLAGS+=	-I${LOCALBASE}/include
 LIBS+=		-L${LOCALBASE}/lib
-USES=		autoreconf gettext-tools iconv
+USES=		autoreconf
 
 PKGMESSAGE=	${WRKSRC}/pkg-message
 SUB_FILES+=	pkg-message
@@ -40,7 +42,6 @@
 
 DEBUG_CFLAGS=	-DDEBUG
 KQUEUE_DESC=	Experimental patch for automatic rescan using kqueue(2)
-NLS_USES=	gettext-runtime
 
 .include <bsd.port.options.mk>
 
Index: Makefile
===================================================================
--- devel/tevent/Makefile	(revision 374080)
+++ devel/tevent/Makefile	(working copy)
@@ -61,7 +61,6 @@
 			%%PKGCONFIGDIR%%/tevent.pc
 
 BUILD_DEPENDS+=		talloc>=2.1.0:${PORTSDIR}/devel/talloc
-RUN_DEPENDS:=		${BUILD_DEPENDS}
 
 CONFIGURE_ARGS+=	--bundled-libraries=!talloc
 
